src/common/exif.cc

```diff
@@ -1170,7 +1171,9 @@ static bool _exif_decode_exif_data(dt_image_t *img, Exiv2::ExifData &exifData)
         { 0.9904476, -0.0071683, -0.0116156, -0.0123712, 1.0155950, -0.0029282, -0.0035635, 0.0067697,
           0.9181569 }, //  Standard light C
         { 0.9212269, -0.0449128, 0.1211620, -0.0553723, 1.0277243, 0.0403563, 0.0235086, -0.0391019,
-          1.6390644 }  // CoolWhiteFluorescent
+          1.6390644 }, // CoolWhiteFluorescent
+        { 0.8662704, -0.0913307, 0.2772741, -0.1090739, 1.0747007, 0.0914149, 0.0551057, -0.0924980,
+          2.5124774 }  // ISO Studio Tungsten
```

Comment:
Where does this come from? Which temperature is ISOStudioTungsten? What is the reference for that?

Reply:
ISO Studio Tungsten is 3200K, as defined in the DNG SDK. The xy (and therefore XYZ) is calculated using the logic in the DNG SDK, and then adaptation using Bradford matrix just like other examples on http://www.brucelindbloom.com/Eqn_ChromAdapt.html

Reply:
OK, maybe worth adding a comment about that, because above it is stated that all matrices come directly from http://www.brucelindbloom.com website.

Reply:
Done